src/components/Notifications.tsx

```diff
@@ -33,6 +33,7 @@ export const Notifications = ({ user }: NotificationProp) => {
         user_email: user.email,
         language: user.lang,
         user_id: user.idUser,
+        filter: `${user.email}`,
```

Comment:
The email is already present, are you sure that sending it again is necessary for filtering?

If in the future you want to filter by another concept, for example, show some messages only for free users, should you change this value? Maybe there is a kind of tag format for this field, for example: `filter: "email:amoschini@makingsense.com;accountType:free`

Reply:
![image](https://user-images.githubusercontent.com/1157864/222755521-324f4b60-d360-4913-aaa7-bf54424b25e3.png)

Reply:
Sure, you can do it adding other filter, for example: filter: `${user.email};${user.accountType}` or something like that, separated by semicolons.

Reply:
> Sure, you can do it adding other filter, for example: filter: `${user.email};${user.accountType}` or something like that, separated by semicolons.

If it is the case, you should set a kind of identifier to avoid mixing pears with apples: `email:${user.email};account_type:${user.accountType}`

Reply:
it's a good idea, I will change it!

Reply:
But, if you only need to filter by user email, probably this `filter` field is not necessary and `user_id` field is enough.

Reply:
it worked with the identifier!

I think it's necessary to add `filter` field if I need to filter by email. In that case `user_id` isn't enough. `user_id` is only used for filter by ID

<img width="687" alt="Captura de Pantalla 2023-03-03 a la(s) 10 31 32 a  m" src="https://user-images.githubusercontent.com/84402180/222760757-e01afbce-120c-4f9c-b7a2-4c6bfc1837f1.png">

Reply:
> is only used for filter by ID

For that reason, I am proposing to use the email as `user_ID` in place of our internal numeric ID.